Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This Dockerfile creates a production release image for the project using crt release flow.
-FROM alpine:3.13 as  default
+FROM alpine:3 as  default
```

Comment:
I'm curious if this was intentionally updated? I think there has been issues in the past upgrading point releases of Alpine, so we might want to stay pin to `3.13`.

Reply:
Yes, this was intentionally updated to allow `alpine:3` to pick up the latest + cleanest version the alpine image. Security scanned `alpine:13` and found some known CVEs. If the goal is to be more prescriptive here, Security scanned `alpine:15` (which is latest + most secure) to be clean of any vulnerabilities. I can absolutely make that change.

Reply:
Ok, makes sense. I did a little searching around Slack to see if there was any reason why we are pinned to `3.13` and I didn't find anything. However, we may want to do this as a separate change considering it could have ramifications that are outside the scope of security scanning. It's up to you though. 

Also, I see comments [like this](https://github.com/hashicorp/consul-terraform-sync/pull/545#discussion_r766927398) from other PRs and am realizing that this is not a new topic for you :) 

To be clear and reiterate, I think this is probably a safe change, but it's your call as to whether we include it here vs in a separate PR.

Reply:
I think it'd be ok to keep this change in this PR and haven't seen any ramifications with this change among other products. Not to be over-generalizing, but to agree with you that this is a safe change :D